modules/ROOT/pages/configuration-schema-db-security.adoc

```diff
@@ -29,9 +29,25 @@ This page introduces the <<put_db-_role-name,Create or Update a Role>> and <<put
 
 It also includes a JSON-data model, you can use to build your request bodies. See: <<lbl-schema>>
 
+[#lbl-upsert-role]
+== Create or Update a Role
+
+
+----
+PUT /{db}/_role/{name}
+----
+
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_role-name-description,leveloffset=-2]
+
+=== Parameters
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_role-name-parameters,leveloffset=-2]
+
+=== Responses
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_role-name-responses,leveloffset=-2]
+
```

Comment:
The parameters in https://preview.docs-test.couchbase.com/docs-sync-gateway-DOC-13345-improve-user/role-documentation/sync-gateway/current/configuration-schema-db-security.html#lbl-upsert-role

are wrong because these document are semi auto generated and I believe we do not know how to update them (cc @osfameron).

This should be dropped to link to the redocly rest api documentations https://docs.couchbase.com/sync-gateway/current/rest_api_admin.html#tag/Database-Security/operation/post_db-_role-